sdk/state/state.go

```diff
@@ -37,6 +38,53 @@ type Channel struct {
 	latestUnauthorizedCloseAgreement CloseAgreement
 }
 
+type CloseState int
+
+const (
+	CloseStateNone         CloseState = iota
+	CloseStateClosing                 // latest declTx is submitted
+	CloseStateNeedsClosing            // an earlier declTx is submitted
```

Comment:
I think this state's name is deceiving, and a little odd that it describes something that needs to happen rather than describes the channel itself. The state is closing, but it is closing with out-of-date state.
```suggestion
	CloseStateClosingWithOutdatedState
```

Reply:
https://github.com/stellar/experimental-payment-channels/pull/232/commits/3cf217ac37068c9eb1897f5a6961e02429cfa6b6